src/EnergyPlus/GroundHeatExchangers.cc

```diff
@@ -741,9 +722,9 @@ BuildAndGetResponseFactorsObjectFromSingleBHs(EnergyPlusData &state, std::vector
 
 //******************************************************************************
 
-void SetupBHPointsForResponseFactorsObject(std::shared_ptr<GLHEResponseFactors> &thisRF)
+void SetupBHPointsForResponseFactorsObject(const std::shared_ptr<GLHEResponseFactors> &thisRF)
```

Comment:
[src/EnergyPlus/GroundHeatExchangers.cc:744]:(style),[constParameter],Parameter 'thisRF' can be declared as reference to const